src/installer/managed/Microsoft.NET.HostModel/Bundle/TargetInfo.cs

```diff
@@ -33,21 +33,21 @@ public TargetInfo(OSPlatform? os, Architecture? arch, Version targetFrameworkVer
         {
             OS = os ?? HostOS;
             Arch = arch ?? RuntimeInformation.OSArchitecture;
-            FrameworkVersion = targetFrameworkVersion ?? net60;
+            FrameworkVersion = targetFrameworkVersion ?? new Version(6, 0);
```

Comment:
```suggestion
            FrameworkVersion = targetFrameworkVersion ?? Environment.Version;
```

same default is used in https://github.com/dotnet/runtime/blob/8fd74f342695704cbb28cb39a47d6f456acc6bbd/src/libraries/Common/tests/StaticTestGenerator/Program.cs#L202-L205

Reply:
Thanks @kasperk81